src/ape_ethereum/ecosystem.py

```diff
@@ -108,25 +82,19 @@ class StaticFeeTransaction(BaseTransaction):
     Transactions that are pre-EIP-1559 and use the ``gasPrice`` field.
     """
 
-    gas_price: int = None  # type: ignore
-    type: TransactionType = TransactionType.STATIC
+    gas_price: Optional[int] = Field(None, alias="gasPrice")
+    max_priority_fee: Optional[int] = Field(None, exclude=True)
```

Comment:
Looking at this, I realize that `TransactionAPI` might be over-specified

Reply:
Add an issue to revisit Ecosystem Api and associated types

Reply:
https://github.com/ApeWorX/ape/issues/528